packages/slate-plugins/src/elements/align/withAlign.ts

```diff
@@ -0,0 +1,25 @@
+import { ReactEditor } from 'slate-react';
+import { wrapAlign } from './transforms';
+
+export const withAlign = () => <T extends ReactEditor>(editor: T) => {
+  const { insertData, insertText } = editor;
+
+  editor.insertText = (text) => {
+    if (text) {
+      wrapAlign(editor, text);
+    } else {
```

Comment:
Why? 🙂

Reply:
I got 'inspired' by the LinkPlugin, this might be the result :-P